src/api/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './CommentList.css';
+import Comment from '../Comment/Comment';
+
+const CommentList = ({ comments }) => (
+  comments.map(comment => (
+    <div className="comments-list" key={comment.id}>
+      <Comment {...comment} />
+    </div>
+  ))
+);
+
+CommentList.propTypes = {
+  comments: PropTypes.arrayOf({
```

Comment:
```suggestion
  
comments: PropTypes.arrayOf(
  PropTypes.shape({
    id: PropTypes.number.isRequired,
  }),
).isRequired,
```
`comments` is an array of objects. You should describe the shape of it